### DISCUSSION TITLE
[Advanced Matching] Is there a way to log user's basic info?

### DISCUSSION THREAD

**Discussion by SlickSlime** [oddbit/flutter_facebook_app_events#196] [Q&A] [answered]
I want to setup Advanced Matching in the app so - I want to know if there a way to log user's basic info (name, email, phone, DOB, gender etc) with this package? In `lib/facebook_app_events.dart` I see method to `clearUserData` method to clear user's data but don't see method to set these values.

Reference for setting up advanced matching on native app: https://developers.facebook.com/docs/app-events/advanced-matching#use-the-facebook-sdk

**Comment by mapacheverdugo** [comment, marked as answer]
There is an open PR (#204) with that function
